src/runtime/link-accounts-flow.js

```diff
@@ -218,6 +218,15 @@ export class LinkSaveFlow {
     /** @private {?Promise<!../api/subscriptions.SaveSubscriptionRequest>} */
     this.requestPromise_ = null;
 
+    /** @private {?Promise} */
+    this.linkedPromise_ = null;
+
+    /** @private {?Promise} */
+    this.confirmPromise_ = null;
+
+    /** @private {?Promise} */
+    this.completePromise_ = null;
```

Comment:
Is this promise actually used anywhere?

Reply:
actually we want this to be called only once
this.dialogManager_.completeView(this.activityIframeView_);
 I thought a completePromise looks nice and can be used for testing to ensure a dialog was completed. I ended not needing in the tests though.

Reply:
FWIW, I think calling `completeView` numerous times is fine. Thought it's ok to have a check here as well. I was just reacting to a null promise which is generally atypical.

Reply:
Addressed in the next patch.